### ISSUE TITLE
can not add user 

### ISSUE THREAD

**Issue by Pancongwen** [FuQiFeiPian/ssh-rpc-agent#4]
## Project version
* tag: 0.0.1
* arch: amd64

## Expected behavior
* I can add a new user named 'f'.
* Let me input new password and other infomation

## task file
```json
[
  {
    	"Topic": "Add new user",
    	"Tasks": [
    		"sudo adduser f"
      	]
    }
]
```

## Step to reproduce
```shell
$ ./ssh-rpc-agent-amd64 --tf task.json --mf machine
You have specify 1 tasks on 1 machines
========================================
Add new user
========================================

[sudo] password for pan: Adding user `f' ...
Adding new group `f' (1014) ...
Adding new user `f' (1009) with group `f' ...
The home directory `/home/f' already exists.  Not copying from `/etc/skel'.
Enter new UNIX password: Retype new UNIX password: passwd: Authentication token manipulation error
passwd: password unchanged
Use of uninitialized value $answer in chop at /usr/sbin/adduser line 591.
Use of uninitialized value $answer in pattern match (m//) at /usr/sbin/adduser line 592.
Try again? [y/N] Changing the user information for f
Enter the new value, or press ENTER for the default
	Full Name []: 	Room Number []: 	Work Phone []: 	Home Phone []: 	Other []: Use of uninitialized value $answer in chop at /usr/sbin/adduser line 621.
Use of uninitialized value $answer in pattern match (m//) at /usr/sbin/adduser line 622.
Is the information correct? [Y/n] Connection to 192.168.21.132 closed.
```

**Comment by dorrywhale** [comment]
I have fixed this issue in commit [f790e83](https://github.com/FuQiFeiPian/ssh-rpc-agent/commit/f790e832329efa6896237e76cae4820dd90dbb85) by add `-l` option for bash, and removing the trick for avoiding typing sudo password, which is not the appropriate design at the first place.

**Comment by Pancongwen** [comment]
The new commit works, thank you!
